OpenRA.Game/Sync.cs

```diff
@@ -55,6 +57,11 @@ internal static int Hash(ISync sync)
 			{ typeof(Target), ((Func<Target, int>)HashTarget).Method },
 		};
 
+		public static void Initialize()
+		{
+			debugSettings = Game.Settings.Debug;
```

Comment:
Btw, I'm also not sure if this really gives us any benefits.

Reply:
it saves one or two lookups i.e. of settings and debug each call. but the saving will trivial. so i removed debugSettings.